explorer/templates/explorer/play.html

```diff
@@ -13,33 +13,38 @@
 <div class="row">
     <div id="query_area" class="col-md-12">
         <h2>Playground</h2>
-        <p>The playground is for experimenting and writing ad-hoc queries. By default, nothing you do here will be saved.</p>
+        <p>The playground is for experimenting and writing ad-hoc queries. By default, nothing you do here will be
+            saved.</p>
         {% if lag_exists %}
-          <div class="alert alert-warning">Warning! The query results might be outdated since some data was last updated around {{ replication_lag }}.</div>
+        <div class="alert alert-warning">Warning! The query results might be outdated since some data was last updated
+            around {{ replication_lag }}.</div>
         {% endif %}
         </br>
         <p>
-            <b>Note:</b> Starting March 16th, access to the <b>phone_number</b> column has been removed for the following
+            <b>Note:</b> Starting March 16th, access to the <b>phone_number</b> column has been removed for the
```

Comment:
Can you change its Date to April 15th, so we can identify easily whether we are running old explorer or new